stories/guides/fozzie/fozzie-migration-guide.stories.mdx

```diff
@@ -143,3 +143,20 @@ module.exports = {
 }
 </style>
 ```
+
+<a name="gulp-migration"></a><h2>If you're coming from Gulp</h2>
+
+If your application uses a different build tool other than webpack (see [prerequisites](#prerequisites) above) such as [Gulp](https://gulpjs.com/). This section will cover the steps you need to complete in-order to migrate successfully.
+
+#### Gulp Build Fozzie:
+
+[Gulp build fozzie](https://github.com/justeat/gulp-build-fozzie) is a set of Gulp tasks used across Fozzie modules. If your application uses this to build packages, instead of Webpack, you will need to firstly update to the latest version (v11.0.0). Version 11.0.0 removes a package called eyeglass which essentially creates a shortcut to your `node_modules` folder.
```

Comment:
```suggestion
[Gulp build fozzie](https://github.com/justeat/gulp-build-fozzie) is a (now deprecated) set of Gulp tasks that can be used to build Fozzie modules. If your application uses this to bundle your project, instead of Webpack, you will firstly need to update to the latest version of `gulp-build-fozzie` (v11.0.0). Version 11.0.0 is compatible with the latest versions of fozzie (v9+), removing support for a package called eyeglass which essentially creates a shortcut to your `node_modules` folder.
```